.openpublishing.redirection.json

```diff
@@ -2033,6 +2033,66 @@
         {
             "source_path":"docs/csharp/reference-semantics-with-value-types.md",
             "redirect_url":"/dotnet/csharp/write-safe-efficient-code"
+        },
+        {
+            "source_path": "docs/csharp/quick-starts/arrays-and-collections.md",
+            "redirect_url": "docs/csharp/tutorials/intro-to-csharp/arrays-and-collections",
```

Comment:
typos here and in all other entries of the "redirect_url": `docs` -> `/dotnet` (note the slash to start with)

Reply:
great catch!

Reply:
That is a great catch. Thanks @pkulikov